.github/workflows/build-test-release.yml

```diff
@@ -4,6 +4,10 @@
 
 name: Release
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
```

Comment:
Is there a reason to use `${{ github.event.number }}-${{ github.event.ref }}` instead of `${{ github.ref }}` like the docs mention?

Reply:
Not sure. I copied it from Matplotlib.

Reply:
Well if it's still working for them we can leave it and see if there's any weird discrepancies in the future.